src/site/content/en/blog/bbc-hsts/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: The BBC rolls out HTTP Strict Transport Security (HSTS)
+subhead: |
+  The BBC is rolling out HSTS for their website to improve security and performance. Find out what it means, and how HSTS can help you.
+authors:
+  - jlwagner
+date: 2022-07-25
+description: |
+  The BBC is rolling out HSTS for their website to improve security and performance. Find out what it means, and how HSTS can help you.
+hero: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/BbUZ7XxZ991yp4lpD5Tr.png
+thumbnail: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/ew6tI1HdAi2g3vxQwdfM.png
+alt: The BBC logo set on a black background.
+tags:
+  - blog
+  - performance
+  - security
+---
+
+HTTPS adoption has been steadily increasing in recent years. Per the HTTP Archive's 2021 Web Almanac, [around 91% of all requests for both desktop and mobile were served over HTTPS](https://almanac.httparchive.org/en/2021/security#transport-security). HTTPS isn't just here to stay, it's a necessary prerequisite to use features such as [Service Worker](https://developer.mozilla.org/docs/Web/API/Service_Worker_API) and modern protocols such as [HTTP/2](/performance-http2/) and [HTTP/3](https://www.cloudflare.com/learning/performance/what-is-http3/).
+
+Recently Neil Craig&mdash;a lead technical Architect at the BBC&mdash;tweeted that [HTTP Strict Transport Security (HSTS)](https://developer.mozilla.org/docs/Web/HTTP/Headers/Strict-Transport-Security) is being slowly rolled out for [bbc.com](https://www.bbc.com/). Let's find out what that means for the BBC, and what it could mean for you.
```

Comment:
I follow Neil so should we know his Twitter handle - should we use that?

```suggestion
Recently [Neil Craig](https://twitter.com/tdp_org)&mdash;a lead technical Architect at the BBC&mdash;tweeted that [HTTP Strict Transport Security (HSTS)](https://developer.mozilla.org/docs/Web/HTTP/Headers/Strict-Transport-Security) is being slowly rolled out for [bbc.com](https://www.bbc.com/). Let's find out what that means for the BBC, and what it could mean for you.
```

Also wonder if we should link his tweet since you've referenced it? https://twitter.com/tdp_org/status/1549028675328573440?s=20&t=S5mm1eEqGUbxZ4jfi-SvtA

Maybe link that rather than HSTS and link that later?